cw-orch-daemon/src/core.rs

```diff
@@ -66,34 +70,120 @@ pub const INSTANTIATE_2_TYPE_URL: &str = "/cosmwasm.wasm.v1.MsgInstantiateContra
     If you do so, you WILL get account sequence errors and your transactions won't get broadcasted.
     Use a Mutex on top of this DaemonAsync to avoid such errors.
 */
-pub struct DaemonAsyncBase<Sender: SenderTrait = Wallet> {
+pub struct DaemonAsyncBase<Sender = Wallet> {
     /// Sender to send transactions to the chain
-    pub sender: Sender,
+    pub(crate) sender: Sender,
     /// State of the daemon
-    pub state: DaemonState,
+    pub(crate) state: DaemonState,
```

Comment:
Is there a special reason for this change? Some actions not possible without `sender` or `state`